src/lib/firebase.js

```diff
@@ -12,7 +12,7 @@ const firebaseApp = firebase.initializeApp({
   storageBucket: import.meta.env.VITE_DUMMYGRAM_STORAGEBUCKET,
   messagingSenderId: import.meta.env.VITE_DUMMYGRAM_MESSAGINGSENDERID,
   appId: import.meta.env.VITE_DUMMYGRAM_APPID,
-  measurementId: import.meta.env.VITE_DUMMYGRAM_MEASUREMENTID,
+  // measurementId: import.meta.env.VITE_DUMMYGRAM_MEASUREMENTID,
```

Comment:
is this not needed? just curious

Reply:
Yup, not needed as we won't be using google analytics.

Reply:
cool